.changeset/tender-pianos-love.md

```diff
@@ -0,0 +1,5 @@
+---
+"@preact/signals": minor
+---
+
+Removed difference in behaviour between adapters. Now signals inserted into jsx can display not only text but other jsx as well
```

Comment:
```suggestion
Removed difference in behaviour between adapters, signals that use a JSX value will correctly re-render the whole component rather than attempting the JSX-Text optimization.
```